### PR TITLE
Removes Greytiding

### PR DESCRIPTION
**Pull request by lcass**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Lcass123
tweak: Makes security and command harder to break into.
del: Removes a spare ID.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
Adds extra security to security to you know make it secure, makes command more secure and removes the easy access greytide ID.

### CODE REVIEW

**Comment by lcass** [comment]
Changes to sec, added some more reinforced walls and remade the armoury entrance.


Makes it harder to breach the HOP office through maint.

**Comment by ghost** [comment]
+1

makes it more challenging and makes sense in theory for these secure areas to actually to be secure

**Comment by DaydreamPuppeh** [comment]
+1, should have been changed a while ago

**Comment by Manly-Dwarf** [comment]
The removal of the spare I.D. is totally unnecessary (actually just makes it so engineers will always get the ID instead of anyone else!) 
Sec changes seem nice though

**Comment by ghost** [comment]
there's a spare ID that spawns in the captain's locker still, there was two whole spare IDs!!
i think
+1 btw

**Comment by Maroonenahmoron** [comment]
I'm really in favor since it's just one thermite into captain's office and you have full access to the station right now, it will also make it impossible to just use bluespace crystals from science/teleporter to whizz in

**Comment by yoyobatty** [comment]
Run check plumbing in game by enabling debug verbs then it'll appear under the mapping tab, just to ensure you didn't delete any pipes.

**Comment by ghost** [comment]
yes, please, people always cheesed cap office with thermite

File: code/game/machinery/doors/airlock_types.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	icon = 'icons/obj/doors/airlocks/station/command.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_com
 	normal_integrity = 450
+	security_level = 6
```
Comment (Jujumatic) [inline code/game/machinery/doors/airlock_types.dm:12]: oh man i think this means it'll take some work to open up the airlock to hacking as well... oh well

**Review by Jujumatic — APPROVED** [review]
looks good

File: code/game/machinery/doors/airlock_types.dm (hunk 1 above)
Comment (lcass) [reply code/game/machinery/doors/airlock_types.dm:12]: Yup, no more instantly breaking into command when you know the open key. Same for the armoury.

Comment (JohnGinnane) [reply code/game/machinery/doors/airlock_types.dm:12]: What differences does security level make?

Comment (lcass) [reply code/game/machinery/doors/airlock_types.dm:12]: sec level 6 make it require 6 extra steps with you being electrocuted without gloves.  IE makes hacking take significantly longer .

Comment (Jujumatic) [reply code/game/machinery/doors/airlock_types.dm:12]: so it's like hacking into the vault door

fuck no, fuck that

**Comment by JessHorserage** [comment]
Seems like a great merge, thanks for doing it yo.

**Comment by Manly-Dwarf** [comment]
I dunno guys, seems like it won't do much to stop engineers from just stealing the locker to get what they'd already achieve any way. It just limits the cheesing, doesn't stop it outright.

**Comment by JohnGinnane** [comment]
@Kawaii-Big-Boss can you explain how engineers have more advantage?

File: code/game/machinery/doors/airlock_types.dm (hunk 1 above)
Comment (Carbonhell) [reply code/game/machinery/doors/airlock_types.dm:12]: i'm up for it but make the sprite different like the vault door one,so people will actually understand this change is in effect

**Comment by Carbonhell** [comment]
stopping isn't the purpose here, making it harder is

**Comment by ghost** [comment]
this is the greatest PR in the history of PRs

File: code/game/machinery/doors/airlock_types.dm (hunk 1 above)
Comment (lcass) [reply code/game/machinery/doors/airlock_types.dm:12]: The only sprite affected by this is the command doors and the armoury doors, no need to change the sprite on command doors if all of them are affected

**Pull request merged**